### ISSUE TITLE
App becomes unusable on export

### ISSUE THREAD

**Issue by cal-zone** [Sequel-Ace/Sequel-Ace#744]
Sequel Ace v 3.02
macOS 11.1

I try to export structure/data for a database or selected tables from the app and it complains I haven't selected a destination (should default to desktop and persist to last entered location). Even after selecting a destination, sometimes whatever I selected goes away and I have to select a new one.

Once I finally have a destination for the export file, if I click on export, the app window vanishes. Eventually, it comes back, but then nothing can be clicked on it (except the menu is still functional and the app is not considered "unresponsive" by the OS). The only solution is to quit and start over.

No matter how many times I try this operation and no matter what destination for saving I choose, the outcome is always the same. 

There is no way to export.

**Comment by Kaspik** [comment]
Export dialog was fixed, so I recommend trying the latest beta or wait for another one later today.

Also, please always fill up the pre-filled issue template.

**Comment by cal-zone** [comment]
There was no pre-filled template when I posted this issue.

**Comment by Jason-Morcos** [comment]
> There was no pre-filled template when I posted this issue.

Hmmm how did you get to the issue submission page? When I do so using the "New Issue" button and then "Bug Report", I do get the template.

<img width="1199" alt="Screen Shot 2021-01-04 at 10 41 35" src="https://user-images.githubusercontent.com/10710367/103567919-7ec85680-4e79-11eb-9997-17120fd1731e.png">
<img width="938" alt="Screen Shot 2021-01-04 at 10 41 51" src="https://user-images.githubusercontent.com/10710367/103567924-7ff98380-4e79-11eb-9693-0d2a25d02cb0.png">

**Comment by Jason-Morcos** [comment]
> Export dialog was fixed, so I recommend trying the latest beta or wait for another one later today.
> 
> Also, please always fill up the pre-filled issue template.

@cal-zone As we believe this is fixed, closing this issue. Please let us know if 3.1.0 beta doesn't resolve this issue. If you're uncomfortable testing the beta, the 3.1.0 release should be made available fairly soon and then you can let us know if that doesn't work for you. Thanks!

Beta link:
https://github.com/Sequel-Ace/Sequel-Ace/releases

**Comment by cal-zone** [comment]
> > There was no pre-filled template when I posted this issue.
> 
> Hmmm how did you get to the issue submission page? When I do so using the "New Issue" button and then "Bug Report", I do get the template.

I was not given the menu to select the issue type when I logged this issue. The second issue I logged I did get that menu.

For this issue, I attempted to log a new issue and it prompted me to create a new account. I tried and it told me I had one already (which I thought I did but my browser did not pre-fill any account info, so I opted to try creating a new one). Once I knew that I did have an account already, I went to the log in page instead and there my browser did pre-fill my login. Once logged in, I was redirected to the issue page where the only things it asked me for were the app and os versions.
